src/storage/mvcc/reader.rs

```diff
@@ -229,7 +241,11 @@ impl MvccReader {
                             }
                             return Ok(write.short_value.take());
                         }
-                        return self.load_data(key, write.start_ts).map(Some);
+                        let seek_write = self.write_cursor.as_ref().unwrap().seek_last_time();
+                        // if we use seek in write cf(there are many history versions),
+                        // use get in default cf to make read faster
+                        let seek_data = !seek_write;
+                        return self.load_data(key, write.start_ts, seek_data).map(Some);
```

Comment:
How about
```rust
if seek_write {
    self.default_cf.seek(......);
}
```
So we don't need `get_with_seek` and we can keep `load_data` simple?

Reply:
Here we should consider default_cf.scan_mode before use seek or seek_for_prev,  so maybe ```get_with_seek``` would be better? @hicqu